### ISSUE TITLE
Keep getting some errors in chat.

### ISSUE THREAD

**Issue by FaultyFunctions** [Mr-EmPee/MysticalBarriers#20]
```console
[11:32:12 ERROR]: Could not pass event PlayerInteractEvent to MysticalBarriers v2.3.1
java.lang.NullPointerException: Cannot invoke "org.bukkit.inventory.meta.ItemMeta.getPersistentDataContainer()" because the return value of "org.bukkit.inventory.ItemStack.getItemMeta()" is null
        at MysticalBarriers-2.3.1.jar/io.github.mr-empee.mysticalbarriers.core.items.PluginItem.getVersion(PluginItem.java:32) ~[MysticalBarriers-2.3.1.jar:?]
        at MysticalBarriers-2.3.1.jar/io.github.mr-empee.mysticalbarriers.core.items.PluginItem.isItem(PluginItem.java:28) ~[MysticalBarriers-2.3.1.jar:?]
        at MysticalBarriers-2.3.1.jar/io.github.mr-empee.mysticalbarriers.core.items.BarrierBlockSelectorWand.onClick(BarrierBlockSelectorWand.java:51) ~[MysticalBarriers-2.3.1.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor842.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-2329-803bf62]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:604) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:560) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:555) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:551) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2088) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:47) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:10) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:56) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1581) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:201) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1558) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1551) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1510) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1517) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1362) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:333) ~[purpur-1.21.1.jar:1.21.1-2329-803bf62]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

I am running Mystical Barriers 2.3.1. We currently have 0 barriers set up so I'm not sure what's going on.

**Comment by Mr-EmPee** [comment]
Hi thanks for reporting the bug, I've located the problem but I'm unable to reproduce it on my instance. The error seems to be from clicking while holding an unexpected item, do you happen to know which item is causing the issue?

**Comment by FaultyFunctions** [comment]
Hmm I'm not sure, but I'll do some testing and get back you to. Might take me a while to find out since we have quite a few custom items from Oraxen and Vane on our server.

**Comment by Mr-EmPee** [comment]
Okay, don't worry in the mean time I will release a fix for the specific problem that you reported but I can't guarantee that there won't be other problems given that I don't know the root cause of it.

**Comment by Mr-EmPee** [comment]
Fixed by be967dd, it would be great if there was more info about the root cause. If the same problem persists feel free to reopen this issue @FaultyFunctions
